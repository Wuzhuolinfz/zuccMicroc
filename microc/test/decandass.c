
void main(int n) {
    int x =5;
  //char a ='x';
    x = x+1;
  print("%d",x);
}