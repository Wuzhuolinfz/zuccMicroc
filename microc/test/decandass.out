24 19 1 5 25 13 0 1 1 0 5 12 13 0 1 1 13 0 1 1 11 0 1 1 12 15 -1 13 0 1 1 11 22 21 2